code/modules/mob/living/carbon/slime/items.dm

```diff
@@ -227,16 +227,27 @@ var/list/global/golem_runes = list()
 	unacidable = TRUE
 	layer = TURF_LAYER
 	var/wizardy = FALSE //if this rune can only be used by a wizard or not
+	var/golem_type = "Adamantine Golem"
 
 /obj/effect/golemrune/Initialize()
 	. = ..()
 	START_PROCESSING(SSprocessing, src)
 	announce_to_ghosts()
 	golem_runes += src
+	if(length(golem_runes) == 1)
```

Comment:
Why specifically 1? What happens if there are more than 1?

Reply:
answered in discord